elf/elf.go

```diff
@@ -257,7 +322,21 @@ func elfReadMaps(file *elf.File) (map[string]*Map, error) {
 			mapCount := len(data) / C.sizeof_struct_bpf_map_def
```

Comment:
> Could you check if `len(data)` is really a multiple of `C.sizeof_struct_bpf_map_def`?

Good point. Otherwise users get a misleading `relocation error` later as the map cannot be found in `b.maps`.

> Actually, since we use the map name included in the name of the ELF section (e.g. `SEC("maps/tcp_event_ipv4")`), we cannot have several `struct bpf_map_def` in one ELF section because we would not know how to distinguish their names.

I'll remove support for several map definitions per `SEC`. We can add it back later based using the symbol name.